app/session_test.go

```diff
@@ -33,7 +40,18 @@ func TestCache(t *testing.T) {
 
 	rkeys := th.App.Srv.sessionCache.Keys()
 	if len(rkeys) != len(keys)-1 {
```

Comment:
nit: you can simplify the code replacing the checks and the `t.Fatal` calls with [require](https://godoc.org/github.com/stretchr/testify/require)

```suggestion
	require.Lenf(t, rkeys, len(keys)-1, "should have one less: %d - %d != 1", len(keys), len(rkeys))
```

Reply:
Great! I didn't know about `require` and it looks much better. I went ahead and changed a few more within the test to improve readability, so I hope that's ok!